src/category_theory/abelian/basic.lean

```diff
@@ -161,11 +161,7 @@ lemma mono_of_zero_kernel (R : C)
 non_preadditive_abelian.mono_of_zero_kernel _ _ l
 
 lemma mono_of_kernel_ι_eq_zero (h : kernel.ι f = 0) : mono f :=
-begin
-  apply mono_of_zero_kernel _ (kernel f),
-  simp_rw ←h,
-  exact is_limit.of_iso_limit (limit.is_limit (parallel_pair f 0)) (iso_of_ι _)
-end
+mono_of_kernel_zero h
```

Comment:
Are these lemmas identical, suggesting one should be deleted?

Reply:
The lemmas aren't identical, one is a special case of the other.